### DISCUSSION TITLE
How do you use the Win64 version release?

### DISCUSSION THREAD

**Discussion by ghost** [rhasspy/piper#379] [Q&A]
I read the help, I follow the steps and I placed my voices at the root of the folder.

The next call doesn't do anything:
`piper.exe TEXT.txt --model es_ES-sharvard-medium.onnx --output_file TEXT.wav`

My folder contents are these:
![pn0J8PF](https://github.com/rhasspy/piper/assets/11261232/c251e1ef-b0e2-47e6-866f-f5dcc02b6369)

No warning, no error messages, nothing is generated.
Thank you.

**Comment by colbec** [comment]
The `piper` command calls for 

> TEXT.txt

 but the folder listing only shows 

> TEXTO.txt?
